cmd/tendermint/commands/rollback.go

```diff
@@ -37,6 +37,10 @@ application.
 func RollbackState(config *cfg.Config) (int64, []byte, error) {
 	// use the parsed config to load the block and state store
 	blockStore, stateStore, err := loadStateAndBlockStore(config)
+	defer func() {
```

Comment:
Is it safe to close these if `err != nil`?

Reply:
yeah might make sense to do the defer afterwards